src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/ManagementRequestSenderService.java

```diff
@@ -99,27 +101,32 @@ public Integer sendAuthCertDeletionRequest(byte[] authCert) throws
         }
     }
 
+    /**
+     * Sends a client register request as a normal X-Road message
+     * @param securityServer the security server id
```

Comment:
javadoc is not in sync with current interface

Reply:
Fixed